app/config/templates/site.php

```diff
@@ -183,13 +183,15 @@ function getFramework(string $frameworkEnum, array $overrides)
         'variables' => []
     ],
     [
-        'key' => 'lynx-starter',
-        'name' => 'Lynx Starter',
-        'tagline' => 'Sample application built with Lynx, a cross-platform framework focused on performance.',
+        'key' => 'playground-for-lynx',
+        'name' => 'Lynx playground',
+        'tagline' => 'A basic Lynx website without Appwrite SDK integration.',
+        // When we add Lynx with Appwrite SDK, use following tagline for it:
+        // 'tagline' => 'Sample application built with Lynx, a cross-platform framework focused on performance.',
```

Comment:
Does this need to be uncommented?

Reply:
I am keeping on pupose because I like the copy, and it's consistent with other taglines for starters.

But right now  we dont have starter, we only hve playground. But I can imagine we add starter too, soon. So keeping the comment for that moment, so we dont need to think about tagline copy again.